src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -12,8 +12,25 @@
                 i18n>Grafana Dashboard doesn't exist. Please refer to
   <cd-doc section="grafana"></cd-doc> on how to add dashboards to Grafana.</cd-alert-panel>
 
+  <cd-alert-panel type="info"
+                  *ngIf="!panelLoaded && !isChrome"
+                  i18n>The embedded Grafana Dashboard was blocked due to a certificate error, please click <a [href]="grafanaSrc"
+                     target="_blank"
+                     noopener
+                     noreferrer>here</a> to accept the security warning then reload this page</cd-alert-panel>
+
+
+  <cd-alert-panel type="info"
+                  *ngIf="isChrome"
+                  i18n>Hi, if the panel below has not loaded please click <a [href]="grafanaSrc"
```

Comment:
Is there a reason for this different style of displaying message on Chrome and firefox. I don't think we've ever tried to greet the user for an info alert before in the dashboard AFAIK. I think its good but can we agree here on a specific style of message on firefox and chrome?

Reply:
Oh ok

Reply:
Hi @nizamial09, @epuertat 

I tried the other approach of making a GET request to the grafanaSrc/url of the panels and it does get blocked. Both firefox and chrome throw an "unkown error".  I can't figure out why the error is unknown and not more descriptive like SSL error. Chrome and curl does throw a hint in the console when the error is logged suggesting this.

firefox:
![firefox](https://user-images.githubusercontent.com/15856785/166216115-8be98d98-7e85-420e-a626-37e8a356b336.png)
chrome
![chrome](https://user-images.githubusercontent.com/15856785/166216241-fcc68194-f433-4a54-bd20-eeeccf407f08.png)
curl:
![curl](https://user-images.githubusercontent.com/15856785/166216309-7502625c-abe9-4cc6-93e3-00aab2d7fca6.png)

My thought was the request would return a webpage of the panel which is the same as when embedding in the iframe directly